azext_iot/tests/central/test_iot_central_devices_int.py

```diff
@@ -318,25 +318,29 @@ def test_central_edge_children_methods(self):
             self.app_id, device_id, child_id
         )
         self.cmd(command, api_version=ApiVersion.v1_1_preview.value)
+        # wait for relationship to be established
+        time.sleep(10)
 
         command = "iot central device edge children list --app-id {} -d {}".format(
             self.app_id, device_id
         )
         # Use api-version 1.0 to also test interoperability
         children = self.cmd(
-            command, api_version=ApiVersion.v1.value
```

Comment:
Will we have a gap in key test coverage by setting the API version here from v1 to v1.1 preview?

Reply:
same as above